<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowCameraWidget</class>
 <widget class="QWidget" name="ShowCameraWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>摄像头</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_back" native="true">
     <widget class="ShowVideoWidget" name="widget_video" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>181</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget_control" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>361</width>
        <height>241</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="comboBox_videoDevice">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>32</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">
/*下拉框*/
QComboBox {
    font-family:&quot;Microsoft YaHei UI&quot;;
    font-size:20px;

    color:#ffffff;
    background-color:#2e3038;

    border-radius:4px;
    border:none;
	padding-left:10px;


	padding-right:30px;
}



QComboBox QAbstractItemView 
{
     outline:0px;
     border:none;
	color:#ffffff;
	background-color:#2e3038;
border-radius:4px;
font-size:20px;

}

QComboBox::drop-down {
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 15px;
     background:transparent;
}
/*图标*/
QComboBox::down-arrow {
    subcontrol-origin: border;
    subcontrol-position:right;

    padding-right:10px;

    width:23px;
    height:25px;
	image: url(:/image/Dropdown_btn.png);
}

QComboBox QAbstractItemView::item
{
height: 48px;
    background-color:#2e3038;
    border:none;
    color:#ffffff;
}

QComboBox QAbstractItemView::item:hover
{
    background-color:#4b5cc4;
    color:#ffffff;
}
</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>36</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_close">
            <property name="minimumSize">
             <size>
              <width>26</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>26</width>
              <height>26</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>关闭</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{ 
image: url(:/image/close_2_normal.png);
border-radius:0px; 
} 
QPushButton:hover{ 
image: url(:image/close_normal.png);
background-color:rgb(255,80,6); 
border-radius:12px; 
} 
QPushButton:pressed{ 
image: url(:image/close_hover.png);
border-radius:0px; 
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShowVideoWidget</class>
   <extends>QWidget</extends>
   <header>Widget/video/ShowVideoWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
